src/mtev_conf.c

```diff
@@ -57,6 +64,10 @@
 #include "mtev_str.h"
 #include "mtev_thread.h"
 
+#if (__STDC_VERSION__ >= 201112L)
```

Comment:
What does this do here?

Reply:
_Static_assert isn't defined before c11.